source/nordic-sdk/components/softdevice/s130/headers/nrf_svc.h

```diff
@@ -49,7 +49,7 @@
   { \
     __asm( \
         "svc %0\n" \
-        "bx r14" : : "I" ((uint32_t)number) : "r0" \
```

Comment:
@metc I am not sure the `uint16_t` fix works anymore. I replaced the `uint32_t` by `uint16_t` in the file `nrf_svc.h` and I got the same warnings.